src/verify.c

```diff
@@ -773,7 +773,7 @@ static int find_unsafe_to_delete(const void *a, const void *b)
 	B = (struct file *)b;
 
 	ret = strcmp(A->filename, B->filename);
-	if (ret) {
+	if (ret != 0) {
```

Comment:
This doesn't look like a bug to me, just a style change. What was the incorrect behavior? Do we need tests for that?

Reply:
I thought I remember in another PR you had added comments mentioning that when the variable was an integer we needed to compare it to an integer and not just use the `variable`, `!variable`, that was valid for pointers and NULL pointers. But I may be getting confused, if this is not the case then we can remove this commit

Reply:
I'm fine with both. You can change that if you prefer, but it's not a bug.
One thing we need to avoid is mixing ints, bools, enums, pointers in comparison. That's important to be careful

Reply:
I'll remove the commit, I bought there had been a discussion about that before but I'm probably mistaken